backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationInitializingService.java

```diff
@@ -132,20 +159,47 @@ public void initAll() {
       if (!userProbe.isPresent()) {
         _apiUserRepo.save(new ApiUser(identity.getUsername(), identity));
       }
-      OrganizationRoleClaims authorization = user.getAuthorization();
+      DemoAuthorization authorization = user.getAuthorization();
       if (authorization != null) {
-        OrganizationRole role =
-            authorization.getEffectiveRole().orElse(OrganizationRole.getDefault());
-        LOG.info(
-            "User {} will have role {} in organization {}",
-            identity.getUsername(),
-            role,
-            authorization.getOrganizationExternalId());
+        Set<OrganizationRole> roles = authorization.getEffectiveRoles();
```

Comment:
I don't think getEffectiveRoles is the right idea here: we should respect the roles set in the configuration, unless we're going to do some configuration-specific transform like adding NO_ACCESS to every user.
```suggestion
        Set<OrganizationRole> roles = authorization.getGrantedRoles();
```

Reply:
that's what happens inside `LiveOktaRepository`!